t/36-utils.t

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
+
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+
+subtest NamedIOSelect => sub {
+    use Utils::NamedIOSelect;
```

Comment:
This will be used at compile time in any case so just move it at the top as usual.

Reply:
:+1: